src/components/ShareResultsModal/ShareResultsModal.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Modal from 'react-responsive-modal';
```

Comment:
What are the benefits of this library? We already have a modal here: https://github.com/Constructor-io/constructorio-ui-quizzes/blob/main/src/components/SessionPromptModal/SessionPromptModal.tsx

Does this library handle some fancy stuff?

Reply:
Mostly just modal placement and also handling clicks outside of the modal itself 😄 I can remove this though if you feel strongly about not using this package. Sorry, I should've checked first on the strict requirement to keep bundle sizes small 😅

Reply:
Removed modal package 👍 ad9ec88